pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1179,6 +1179,9 @@ def setup_config_options(cls, install_directory_help=None, **kwargs):
         cls.buildFpgaKernels = cls.add_bool_option("build-fpga-kernels", show_help=True, _allow_unknown_targets=True,
                                                  only_add_for_targets=fpga_targets,
                                                  help="Also build kernels for the FPGA.")
+        cls.buildFettKernels = cls.add_bool_option("build-fett-kernels", show_help=True, _allow_unknown_targets=True,
```

Comment:
I've been trying to gradually clean up the inconsistent naming to move to underscores since that's the python default. I'm happy to change this as part of the bulk rename commits so no urgent need to update this pr.

Reply:
Alternatively it'd be nice if cheribsd-riscv64-purecap extended from cheribsd-riscv64 (and cheribsd-riscv64-nocheri would be where you put things specifically for _that_).... :D